driver-adapters-wasm/pg-node-basic-wasm-edge-import/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "pg-node-basic-wasm-edge-import",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest index.test.mjs"
```

Comment:
This (and some file renames, change from require to import) was needed to be able to run the tests, modelled after the other `esm` project under `driver-adapters`.